BrowserServicesKit/Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -688,6 +693,12 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         do {
             try await load(options: startupOptions)
             Logger.networkProtection.log("Startup options loaded correctly")
+
+#if os(iOS)
+            if (try? await tokenHandler.getToken()) == nil {
+                throw TunnelError.startingTunnelWithoutAuthToken(internalError: nil)
+            }
+#endif
```

Comment:
This is the original check, except wrapped in an iOS check as this problem doesn't occur on macOS. I'd rather play it safe and restore this check for iOS-only for now.